mocks/mock-lighthouse-api/src/main/java/gov/va/vro/mocklh/config/AppConfig.java

```diff
@@ -3,15 +3,22 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import gov.va.vro.mocklh.model.MockBundleStore;
 import gov.va.vro.mocklh.model.MockBundles;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
 import org.springframework.web.client.RestTemplate;
 
 import java.io.IOException;
 
+@Slf4j
 @Configuration
 public class AppConfig {
 
+  private static final String BASE_FOLDER = "mock-bundles";
+  private static final String FOLDER_PATTERN = "classpath:" + BASE_FOLDER + "/*/Condition.json";
```

Comment:
Java/Spring Boot are some strange beasts when trying to read a directory listing from both an IDE/command line and a docker image.

This just works around the problem in the least hacky way I could find.